benchmark/sirun/async_hooks/meta.json

```diff
@@ -0,0 +1,9 @@
+{
+  "name": "async_hooks",
+  "run": "node -r ../monitor .",
+  "variants": [
```

Comment:
Should we do additional variants for different numbers of promises per second?

Reply:
What would be the benefit? Since the only thing the app does is create promises I figured it would be enough to go with a good average.